<?php

Test::add("Create new user", function() {
    try {
        //
        echo "OK\n\n";
    } catch (Exception $e) {
        echo "ERROR: ". $e->getMessage() . "\n\n";
    }
});